Packs/SmartSheetITAdmin/Integrations/Smartsheet_IT_Admin/Smartsheet_IT_Admin.py

```diff
@@ -0,0 +1,516 @@
+                user_extension_data = extension_schema.get(key)
+                if user_extension_data is not None:
+                    smartsheet_user[value] = user_extension_data
+        if not smartsheet_user.get('admin', None):
+            smartsheet_user['admin'] = False
+        if not smartsheet_user.get('licensedSheetCreator', None):
+            smartsheet_user['licensedSheetCreator'] = False
+        return smartsheet_user
+
+    # Builds a new user prisma profile dict with pre-defined keys and custom mapping (for user)
+
+    def build_smartsheet_update_user(self, args, scim, custom_mapping):
+        parsed_scim_data = map_scim(scim)
+
+        smartsheet_user = {
+            # "email": parsed_scim_data.get('email'),
+            "firstName": parsed_scim_data.get('first_name'),
+            "lastName": parsed_scim_data.get('last_name'),
+            "locale": parsed_scim_data.get('locale'),
+            "timeZone": parsed_scim_data.get('timezone'),
+            "alternateEmails": parsed_scim_data.get('alternate_emails'),
+            "title": parsed_scim_data.get('title'),
+            "department": parsed_scim_data.get('department'),
+            "workPhone": parsed_scim_data.get('phone_mobile'),
+            "mobilePhone": parsed_scim_data.get('phone_work'),
+        }
+        if args.get('customMapping'):
+            custom_mapping = json.loads(args.get('customMapping'))
+        elif custom_mapping:
+            custom_mapping = json.loads(custom_mapping)
+
+        extension_schema = scim.get(SCIM_EXTENSION_SCHEMA)
+        if custom_mapping and extension_schema:
+            for key, value in custom_mapping.items():
+                # key is the attribute name in input scim. value is the attribute name of app profile
+                user_extension_data = extension_schema.get(key)
+                if user_extension_data is not None:
+                    smartsheet_user[value] = user_extension_data
+
+        return smartsheet_user
+
+    def build_smartsheet_remove_user(self, args, scim, custom_mapping):
+
+        smartsheet_user = dict()
+        if args.get('customMapping'):
+            custom_mapping = json.loads(args.get('customMapping'))
+        elif custom_mapping:
+            custom_mapping = json.loads(custom_mapping)
+
+        extension_schema = scim.get(SCIM_EXTENSION_SCHEMA)
+        if custom_mapping and extension_schema:
+            for key, value in custom_mapping.items():
+                # key is the attribute name in input scim. value is the attribute name of app profile
+                user_extension_data = extension_schema.get(key)
+                if user_extension_data is not None:
+                    smartsheet_user[value] = user_extension_data
+
+        return smartsheet_user
+
+
+class OutputContext:
+    """
+        Class to build a generic output and context.
+    """
+
+    def __init__(self, success=None, active=None, iden=None, username=None, email=None, errorCode=None,
```

Comment:
I think most of the attributes defined here are not really used anywhere, aren't they?

Reply:
All the attributes here are being use somewhere in the code. You can refer to line 288 and line 291.